src/main/java/nwoolcan/viewmodel/brewery/production/batch/review/BatchEvaluationDTO.java

```diff
@@ -0,0 +1,42 @@
+package nwoolcan.viewmodel.brewery.production.batch.review;
+
+import nwoolcan.model.brewery.production.batch.review.BatchEvaluationType;
+import nwoolcan.model.brewery.production.batch.review.EvaluationType;
+import org.apache.commons.lang3.tuple.Pair;
+
+import java.util.Collection;
+
+/**
+ * Mockup evaluation created from the view.
+ * No controls on data integrity.
+ */
+public class BatchEvaluationDTO {
```

Comment:
I think it should be called viewmodel
```
public class BatchEvaluationViewModel {
```

Reply:
There's another `BatchEvaluationViewModel`, this is just an object to pass to the controller to create a new `BatchEvaluation`, it should not be displayed in the view